recipes/ifcfg/meta.yaml

```
@@ -0,0 +1,36 @@
+{% set name = "ifcfg" %}
+{% set version = "0.19" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/ftao/python-{{ name }}/archive/releases/{{ version }}.tar.gz"
+  sha256: "dc7effa7b4d0d4b7ff4c9517693315fa3bde4eb826e185645549b85efb5513fb"
+
+build:
+  number: 0
```

Looks like it can be noarch?
```suggestion
  noarch: python
  number: 0
```